handlers/github_organization_repo_create_v1/handler/init.rb

```diff
@@ -20,22 +20,47 @@ def initialize(input)
       # trailing whitespace)
       @parameters[node.attribute('name').value] = node.text.to_s.strip
     end
+
+    @debug_logging_enabled = ["yes","true"].include?(@info_values['enable_debug_logging'].downcase)
+    @error_handling = @parameters["error_handling"]
   end
 
   def execute()
 
-      url = "https://api.github.com/orgs/#{@info_values['org_name']}/repos?access_token=#{@info_values['access_token']}"
+    # Initialize return data
+    error_message = nil
 
+    begin
+      url = "https://api.github.com/orgs/#{@info_values['org_name']}/repos?access_token=#{@info_values['access_token']}"
+      
       resource = RestClient::Resource.new(url)
 
-      data = {name: @parameters['name'], description: @parameters['description']}.to_json
+      data = {name: @parameters['name'], description: @parameters['description']}
 
-      resource.post(data, {content_type: :json})
+      data['private'] = true if @parameters['visibility'].downcase == "private"
+
+      puts "Calling URL https://api.github.com/orgs/#{@info_values['org_name']}/repos?access_token=XXX" if @debug_logging_enabled
+      response = resource.post(data.to_json, {content_type: :json})
 
       results = <<-RESULTS
       <results>
+        <result name="Response Body">#{escape(response.nil? ? {} : response.body)}</result>
+        <result name="Handler Error Message">#{escape(error_message)}</result>
       </results>
       RESULTS
+    rescue RestClient::Exception => e
+      error_message = e.inspect
```

Comment:
Why `e.inspect` instead of just `e.message`?

Reply:
I have no specific preference for either.  It is a holdover from where I got the original code.  Does .message have the same result as .inspect?

Reply:
Jack, I looked into this more.  The [Ruby Exception doc](https://www.rubydoc.info/gems/rest-client/1.6.1/RestClient/Exception) show that inspect returns "#{message}: #{http_body}".  I think I like that more than just returning the message.

Reply:
OK